pkg/apis/serving/v1beta1/explainer.go

```diff
@@ -26,6 +26,8 @@ import (
 type ExplainerSpec struct {
 	// Spec for ART explainer
 	ART *ARTExplainerSpec `json:"art,omitempty"`
+	// Explainer spec for any arbitrary framework.
+	Model *ModelSpec `json:"model,omitempty"`
```

Comment:
This is currently defined in `predictor.go`. Perhaps we should extract that out?

Reply:
This is actually defined in `model.go` which I renamed (was `predictor_model.go`) so it is more logical that is used in both `explainer.go` and `predictor.go`. In `model.go` the `PredictorExtensionSpec` is used which is defined in `predictor.go` do you think we should extract that out?